src/mainwindow/mainwindow.cpp

```diff
@@ -2960,9 +2960,17 @@ QString MainWindow::getStyleSheetSuffix() {
     return "fritzing";
 }
 
-void MainWindow::addToMyParts(ModelPart * modelPart)
+void MainWindow::addToMyParts(ModelPart * modelPart, QStringList & m_peAlienFiles)
 {
```

Comment:
The m_ prefix should be used for members, not parameters

Reply:
I think, both modelPart and peAlienFiles, should be past as const reference.

Reply:
> The m_ prefix should be used for members, not parameters

No problem, your help is welcome.
The name change is about 'coding style'.. rule of thump is that the m_ prefix is only for variables that are members. This way, when reading the code, one can see if changing the variable only has a local effect, or affects the whole instance.